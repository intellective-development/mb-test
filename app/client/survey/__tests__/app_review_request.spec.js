import * as React from 'react';
import AppReviewRequest from '../app_review_request';

describe('AppReviewRequest', () => {
  it('renders a request when the score prop is 5', () => {
    expect(render(
      <AppReviewRequest score={5} />
    )).toMatchSnapshot();
  });

  it('renders null when the score prop is less than five', () => {
    expect(render(
      <AppReviewRequest score={4} />
    )).toMatchSnapshot();
  });
});
