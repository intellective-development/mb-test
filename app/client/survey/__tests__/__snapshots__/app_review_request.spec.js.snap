// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AppReviewRequest renders a request when the score prop is 5 1`] = `
<div>
  <a
    href="https://itunes.apple.com/us/app/minibar-delivery-wine-liquor/id720850888?mt=8&uo=4&at=11l5XB&ct=minibar"
  >
    Review our app and help spread the word.
  </a>
</div>
`;

exports[`AppReviewRequest renders null when the score prop is less than five 1`] = `null`;
